services/ui-src/src/components/drawers/ReportDrawerDetails.tsx

```diff
@@ -13,12 +13,34 @@ export const ReportDrawerDetails = ({ entityType, drawerDetails }: Props) => {
           <Text sx={sx.detailDescription}>
             {drawerDetails.standardDescription}
           </Text>
-          <Text sx={sx.detailCategoryHeader}>General Category</Text>
-          <Text sx={sx.detailCategory}>{drawerDetails.category}</Text>
+          <Text sx={sx.detailSubtitle}>General Category</Text>
+          <Text sx={sx.detailSubtext}>{drawerDetails.category}</Text>
         </Box>
       );
     case ModalDrawerEntityTypes.SANCTIONS:
-      return <Text sx={sx.detailDescription}>Sanctions TODO</Text>;
+      return (
+        <Box sx={sx.detailBox}>
+          <Heading as="h4" sx={sx.detailHeader}>
+            Intervention type - {drawerDetails.interventionType}
```

Comment:
Not sure if this is a design question, but can we make this a colon instead of a dash? It made the title a bit confusing to me. 

```suggestion
            Intervention type: {drawerDetails.interventionType}
```